examples/flux-source/index.ts

```diff
@@ -0,0 +1,251 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as kubernetes from "@pulumi/kubernetes";
+import { ConfigGroup } from "@pulumi/kubernetes/yaml";
+import * as flux from "@worawat/flux";
+
+const config = new pulumi.Config();
+const deployNamespace = config.get("namespace") || 'default';
+const deployNamespaceList = config.getObject<string[]>("namespaces") || [deployNamespace];
+const operatorVersion = config.get("operator-version") || "v1.9.0";
+const crdVersion = config.get("crd-version") || "v1.9.0";
```

Comment:
```suggestion
const crdVersion = config.get("crd-version") || "v1.10.0";
```

I think we need this to default to 1.10, when the program CRDs were added.